target/linux/ipq40xx/image/Makefile

```diff
@@ -627,6 +627,35 @@ define Device/qcom_ap-dk04.1-c1
 endef
 TARGET_DEVICES += qcom_ap-dk04.1-c1
 
+define Device/p2w_r619ac
```

Comment:
Please move according to alphabetic sorting. Since most of the stuff is shared, please create a shared definition Device/p2w_r619ac_common or similar.